slurm_ops_manager/slurm_ops.py

```diff
@@ -68,6 +68,46 @@ def slurm_component(self) -> str:
         """Return the slurm component."""
         return self._slurm_resource_manager.slurm_component
 
+    @property
+    def fluentbit_config_nhc(self) -> list:
+        """Return Fluentbit configuration parameters to forward NHC logs."""
+        cfg = [{"input": [("name",             "tail"),
+                          ("path",             "/var/log/nhc.log"),
+                          ("path_key",         "filename"),
+                          ("tag",              "nhc"),
+                          ("multiline.parser", "nhc")]},
+               {"multiline_parser": [("name",          "nhc"),
+                                     ("type",          "regex"),
+                                     ("flush_timeout", "1000"),
+                                     ("rule",          '"start_state"', '"/^([\d]{8} [\d:]*) (.*)/"', '"cont"'), # noqa
```

Comment:
The formatting here is kinda tough to read. Is there another way we could format it?

Reply:
This part of the config looks like this in Fluentbit config file:

```
[MULTILINE_PARSER]
    name           nhc
    type           regex
    flush_timeout  1000
    rule           "start_state" "/^([\d]{8} [\d:]*) (.*)/" "cont"
    rule           "cont"        "/^([^\d].*)/"             "cont"
```

Looks reasonable to me. Do you have another suggestion?

Reply:
What if we apply black here?

Reply:
I don't think it's hard to understand. If Heitor (the main maintener here) is ok then we could simply pass it